exercises/perfect-numbers/example.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var Divisors = require('./divisors');
+
+var PerfectNumbers = function() {
+
+}
+
+PerfectNumbers.prototype.classify = function(number) {
+
+  var i, sum, result;
+  var divisors = new Divisors();
+
+  // Check if the input is valid
+  if (number <= 0)
+    return 'Classification is only possible for natural numbers.';
+
+  // Factorize the current number.
+  var divsArray = divisors.getDivisors(number);
+
+  // Sum the factors.
+  sum = 0;
+  for (i = 0; i < divsArray.length; i++) {
+    sum = sum + divsArray[i];
+  }
```

Comment:
Thanks for mentioning that. However this method uses ES6, right? With that arrow function `=>`. I thought that I was limited to ES5, because I tried to use the `import` keyword and the jasmine tests failed.